libraries/AP_Notify/RGBLed.cpp

```diff
@@ -169,7 +169,7 @@ void RGBLed::update_colours(void)
 // at 50Hz
 void RGBLed::update()
 {
-    if (!pNotify->_rgb_led_override) {
+    if (pNotify && !pNotify->_rgb_led_override) {
```

Comment:
This is wrong. You can only get here if you don't have a notify object. All notify devices are only meant to run with notify objects existing and no behavior is guaranteed without it, and this should be reverted.

If you have an example that is hitting this, then the example should be granted a notify instance.